examples/example.cpp

```diff
@@ -89,6 +89,8 @@ int main( int argc, const char *argv[] )
     auto span_1 = trace.CreateSpan("Span1");
     portable_sleep(1);
     span_1.AddAnnotation("Span1Event");
+    span_1.AddBinaryAnnotation("str value", "some value");
+    span_1.AddBinaryAnnotation("boolean value", true);
```

Comment:
zipkin will drop all of these (except a very special case).. probably want to switch to tags to avoid steering new people into this direction.  https://zipkin.io/zipkin-api/#/default/post_spans

Reply:
Sadly i only support version 1, i will add support for version 2 in the coming days. i will open a new issue for it.

Reply:
ok I guess I mean that the api model can be independent of the formatting. if api says "tag" it can still be made into a binary annotation at formatting layer (this is how we do it). So I was noticing as this is a new api, possibly can skip the vocabulary tie to "binary annotation"

Reply:
Its a good point, i will make the changes, thanks!